### PR TITLE
chore: removed liquibase and replaced with sequelize for migrations -…

### PR DESCRIPTION
**Pull request by richard-mohammed-def**
… added first migration for idm update

### CODE REVIEW

File: packages/api/package.json [hunk 1]
```diff
@@ -15,7 +15,9 @@
     "start": "node src/api-service.js",
     "dev": "nodemon -r dotenv/config src/api-service.js",
     "test": "echo \"Error: run tests from root\" && exit 1",
-    "postinstall": "if [ \"${NODE_ENV:-development}\" == \"development\" ]; then npx webpack-cli; else echo no webpack build; fi;"
+    "postinstall": "if [ \"${NODE_ENV:-development}\" == \"development\" ]; then npx webpack-cli; else echo no webpack build; fi;",
+    "database:migration:create": "npx sequelize-cli migration:generate --name migration; echo \"\nPlease change generated file extension from .js to .cjs before running migrate script.\";",
+    "database:migration:migrate": "npx sequelize-cli db:migrate --url \"postgres://${POSTGRES_USER}:${POSTGRES_PW}@${POSTGRES_HOST}/${POSTGRES_DB}\""
```
Comment (StuAA78) [inline packages/api/package.json:20]: Can we add a script that calls `db:migrate:undo`? We're writing a `down` method in our migrations so we may as well add a script to use it 🙂

**Review by StuAA78 — CHANGES_REQUESTED** [review]

File: packages/api/start.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+npm install -g sequelize-cli;
+npm run database:migration:migrate;
+node src/api-service.js;
```
[the remark this answers is not in the document]
Comment (richard-mohammed-def) [reply packages/api/start.sh:3]: We certainly can. Will change.

File: packages/api/package.json (hunk 1 above)
Comment (richard-mohammed-def) [reply packages/api/package.json:20]: Sure thing

File: packages/api/start.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (richard-mohammed-def) [reply packages/api/start.sh]: Changes made

**Review by StuAA78 — APPROVED** [review]
Nice one 👍

**Pull request merged**